pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -31,7 +34,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-jBwyIyiWeqNmOnxmVOr7c4oMWwHElEjM25sShhTMi78=";
   };
 
-  nativeBuildInputs = [ yarn fixup-yarn-lock nodejs copyDesktopItems makeWrapper ];
+  nativeBuildInputs = [
+    yarn
+    fixup-yarn-lock
+    nodejs
+    makeWrapper
+  ] ++ lib.optionals (!stdenv.isDarwin) [
```

```suggestion
  ] ++ lib.optionals (stdenv.isLinux) [
```

Be explicit about Linux